src/components/RecommendedFile.vue

```diff
@@ -78,6 +79,21 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		timestamp: {
+			type: Number,
+			default: null,
```

Comment:
Can this ever be null? If not its easier to require it.
```suggestion
			required: true,
```